src/tests/performance/connections.rs

```diff
@@ -14,34 +17,36 @@ enum PeerEvent {
     ReadError(std::io::Error),
 }
 
-#[derive(Default, Debug)]
-struct ConnectionStats {
-    /// count of successfuly handshaken connections
-    pub success: u16,
-    /// count of connections rejected pre-handshake
+#[derive(Tabled, Default, Debug, Clone)]
+struct Stats {
+    #[header(" N ")]
+    pub max_peers: u16,
+    #[header(" M ")]
```

Comment:
Let's use the full name as a header (would also be more consistent with our other tables)?

Reply:
Done